doc/Media2.xml

```diff
@@ -1005,6 +1005,7 @@
         </itemizedlist>
         <para>TheVideoEncoderConfiguration structure also contains multicast parameters. </para>
         <para>If the whole RateControl parameter structure is missing the current state of rate control is undefined and vendor specific. A device that supports disabling of the rate control mechanisms shall reflect that by omitting the RateControl element when disabled; otherwise it shall return the current values used for RateControl. If RateControl is missing, the respective options define whether a RateControl element can be (re-)added.</para>
+        <para>The resolution's width and height are based on the Bounds field of the VideoSourceConfiguration. Therefore, if there is a 90-degree rotation, the width will represent the height of the encoded video, and the height will represent the width of the encoded video.</para>
```

Comment:
Concise text suggestion
- **Therefore, if video source undergoes a 90-degree rotation, The height and width of the encoded video will be represented by the video source width and height, respectively.**

Reply:
> Concise text suggestion
> 
> * **Therefore, if video source undergoes a 90-degree rotation, The height and width of the encoded video will be represented by the video source width and height, respectively.**

@bsriramprasad 
 In my opinion, your suggestion _"The height and width of the encoded video will be **represented** by the video source width and height"_ is a bit ambiguous and doesn't seem to convey the behavior as expected.